shared/common-adapters/emoji.desktop.js

```diff
@@ -10,9 +10,32 @@ import type {Props} from './emoji'
 
 const backgroundImageFn = (set: string, sheetSize: string) => emojiSet
 
+// Workaround for bug where if an invalid emoji name is supplied to Emoji,
+// a full crash is triggered. Here we check the value between the first
+// pair of colons & make sure it's not a skin-tone modifier and that we
```

Comment:
there's no explicit check that it's a skin-tone modifier. Was that deliberate?

Reply:
that was deliberate, I updated the comment